packages/vue/src/components/molecules/SfImage/SfImage.html

```diff
@@ -1,17 +1,13 @@
-<div
-  class="sf-image"
-  :style="{maxWidth: maxWidth}"
-  @mouseover="hoverHandler(true)"
-  @mouseleave="hoverHandler(false)">
+<div class="sf-image" :style="{maxWidth: maxWidth}" @mouseover="hoverHandler(true)" @mouseleave="hoverHandler(false)">
   <transition :name="transition">
     <div v-if="hasOverlay && loaded" class="sf-image__overlay">
       <!--@slot slot for overlay content, if is empty the overlay is off -->
-      <slot/>
+      <slot />
     </div>
   </transition>
-  <img
-    ref="img"
-    loading="lazy"
-    class="sf-image__img"
-  >
+  <img ref="img" loading="lazy" class="sf-image__img lozad" v-if="src" />
+  <picture ref="picture" loading="lazy" class="sf-image__img lozad" v-else>
+    <source ref="pictureSmall" media="(max-width: 767px)">
```

Comment:
por quê não jogar o srcset aqui de uma vez?
```suggestion
    <source srcset="pictures[0].normal.url" media="(max-width: 767px)">
```